.github/workflows/image-scan-analysis.yml

```diff
@@ -0,0 +1,133 @@
+name: Security CI
```

Comment:
Seems unrelated to the PR

Reply:
I added  action to scan images before deployment